apps/dot/dot.yml

```diff
@@ -0,0 +1,18 @@
+
+name: Dot
+description: 'A sleek ad-free browser made with Material UI.' 
+website: 'https://getdot.js.org'
+category: 'Web Browser'
+keywords:
+- browser 
```

Comment:
I suspect the CI failure is due to the yaml parser dying here due to lack of indentation in the keywords list on this line and below